oreo_base: {active_range: 2,  
  ang_diff_max: 0.6981317008, 
  ang_diff_min: 0.02635987756,
  ang_vel_diff_max: 0.01, 
  ang_vel_max: 0.6, 
  lin_vel_diff_max: 15, 
  lin_vel_max: 300.0,
  geer_ratio: 43, 
  h_com: 2, 
  horizontal_wheel_counts_per_rev: 176128,
  l_com: 3, 
  left_wheel_counts_per_rev: 176128,
  r_com: 1, 
  right_wheel_counts_per_rev: 176128,
  a_balance: 1.01,
  l_balance: 1, 
  robot_radius: 0.225, 
  t_balance: 1, 
  wheel_diameter: 0.152,
  kv: 5, 
  kw: 0.008, 
  kww: 0.5, 
  vr: 4, 
  wr: 0, 
  ka: 0.0016}
